src/vs/workbench/contrib/comments/browser/commentMenus.ts

```diff
@@ -35,10 +35,6 @@ export class CommentMenus implements IDisposable {
 		return this.getMenu(MenuId.CommentThreadTitleContext, contextKeyService);
 	}
 
```

Comment:
After this is merged should I go in and make the same adoption for the other menus here?

Reply:
Yeah, ideally the `WorkbenchToolbar` or the `MenuWorkbenchToolbar` can be used. They should make it easy to render menus as buttons, context menus is done in this PR